tools/macos/README.md

```diff
@@ -1,8 +1,89 @@
-#docker-machine:
+# How to setup a Mac host for OpenWhisk
 
-* one time setup
+One way to develop or deploy OpenWhisk on a Mac is to use [docker-machine](https://docs.docker.com/machine/install-machine/) which runs the Docker daemon inside a virtual machine accessible from the Mac host.
+
+# Prerequisites
+
+The following are required to build and deploy OpenWhisk from a Mac host:
+
+- [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+- [Docker](https://docs.docker.com/engine/installation/mac/) (including `docker-machine`)
```

Comment:
What version of docker do we recommend? 1.9, 1.10, 1.11  It will be good if its matches the version being used during testing in Travis, In Travis we install like this `sudo apt-get -o Dpkg::Options::="--force-confnew" --force-yes -y install docker-engine=1.9.1-0~trusty`